### PR TITLE
[16.0][MIG] account_cutoff_base: Allow to configure different account for refunds (credit notes)

### PR DESCRIPTION
**Pull request by jbaudoux**
Forward port commit https://github.com/OCA/account-closing/commit/83e9b1c7f9463043414d88374f67c9ee01873b9a

### CODE REVIEW

**Comment by jbaudoux** [comment]
@OCA/accounting-maintainers Could you reopen this PR and merge it ?

cc @rousseldenis

**Comment by jbaudoux** [comment]
@adrienpeiffer Can you merge this one also ?
cc @lmignon

File: account_cutoff_base/models/res_company.py [hunk 1]
```diff
@@ -50,3 +50,13 @@ class ResCompany(models.Model):
         string="Default Account for Prepaid Expense",
         check_company=True,
     )
+    default_accrued_revenue_return_account_id = fields.Many2one(
+        "account.account",
+        string="Default Account for Accrued Revenues Refunds",
+        check_company=True,
+    )
+    default_accrued_expense_return_account_id = fields.Many2one(
```
Comment (adrienpeiffer) [inline account_cutoff_base/models/res_company.py:58]: @jbaudoux Where these fields are used ?

Comment (sbidoul) [reply account_cutoff_base/models/res_company.py:58]: These are used in other, not yet merged modules.

**Review by sbidoul — APPROVED** [review]

**Pull request merged**